runners/google-cloud-dataflow-java/src/test/java/org/apache/beam/runners/dataflow/DataflowRunnerTest.java

```diff
@@ -1001,6 +1007,90 @@ public void translate(
     assertTrue(transform.translated);
   }
 
+  @Test
+  public void testMapStateUnsupportedInBatch() throws Exception {
```

Comment:
These 4 methods be deduplicated into a method like `<StateT> testStateTypeUnsupported(boolean isStreaming, StateSpec<StateT> spec)` though I'm not sure it's worth it.